source/link.hpp

```diff
@@ -38,8 +38,8 @@ class ILink : public Identifiable {
 class Link : public ILink, public std::enable_shared_from_this<Link> {
 public:
     Link(std::weak_ptr<IRoutingDevice> a_from,
-         std::weak_ptr<IRoutingDevice> a_to, std::uint32_t a_speed_gbps = 1,
-         Time a_delay = 0);
+         std::weak_ptr<IRoutingDevice> a_to, std::uint32_t a_speed_mbps = 1,
```

Comment:
From now one we store speed in gbps
```suggestion
         std::weak_ptr<IRoutingDevice> a_to, std::uint32_t a_speed_gbps = 1,
```